src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -100,11 +100,64 @@ static std::string          subscribeContextAvailabilityCollectionName;
 static Notifier*            notifier;
```

Comment:
CNR line missing. Not sure about the wording... but it should say that we have fixed memory errores although based in a workaround and not a definitive solution. Something like this:

```
- Fix: several invalid memory accesses (based in a workaround, not a definitive solution, see issue #2994)
```

Reply:
Fixed in dcf5197